Tasks/PackerBuildV0/task.json

```diff
@@ -56,6 +56,22 @@
             },
             "defaultValue": "builtin"
         },
+        {
+            "name": "isManagedImage",
+            "type": "boolean",
+            "label": "Managed VM disk image",
+            "required": true,
+            "helpMarkDown": "Check if Managed disk image is to be created for auto generated Packer template or if Managed disk image is generated by user provisioned Packer template.",
+            "defaultValue": false
+        },
+        {
+            "name": "managedImageName",
```

Comment:
Isn't this same as generated managed image name? if this is an input, why does it need to be a output as well?

Reply:
Is it because is case of custom template we won't know what is the name of image?

Reply:
is it mandatory to specify image name as input? if we don't specify we can read name from output

Reply:
Managed image name needs to specified in the packer azure managed image template.  The this field is **only required for builit in template**. For user defined templates this name will be in the template.

Reply:
okay. makes sense.